specification/riskiq/data-plane/Microsoft.Easm/preview/2022-09-01-preview/easm.json

```diff
@@ -0,0 +1,5550 @@
+        "cookieDomain": {
+          "type": "string"
+        },
+        "cookieExpiryDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "cookieName": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "firstSeen": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "lastSeen": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "recent": {
+          "type": "boolean"
+        }
+      },
+      "type": "object"
+    },
+    "Cve": {
+      "properties": {
+        "cvss3Summary": {
+          "$ref": "#/definitions/CVSS3Summary"
+        },
+        "cvssScore": {
+          "format": "float",
+          "type": "number"
+        },
+        "cweId": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectionPageResponse": {
+      "properties": {
+        "content": {
+          "description": "The items in the current page of results.",
+          "items": {
+            "$ref": "#/definitions/DataConnectionResponse"
+          },
+          "type": "array"
+        },
+        "first": {
+          "description": "Whether or not this is the first page of results.",
+          "type": "boolean"
+        },
+        "last": {
+          "description": "Whether or not this is the last page of results.",
+          "type": "boolean"
+        },
+        "nextLink": {
+          "description": "The link to access the next page of results.  Not set if at the end of the result set.",
+          "type": "string"
+        },
+        "number": {
+          "description": "The page number requested.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "numberOfElements": {
+          "description": "The number of items in the current page of results.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "size": {
+          "description": "The page size requested.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "totalElements": {
+          "description": "The total number of items available in the full result set.",
+          "format": "int64",
+          "type": "integer"
+        },
+        "totalPages": {
+          "description": "The total number of pages available in the full result set.",
+          "format": "int32",
+          "type": "integer"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectionRequest": {
+      "properties": {
+        "connectionString": {
+          "description": "The connection string for the data connection.",
+          "type": "string"
+        },
+        "kind": {
+          "description": "The type of the data connection.",
+          "enum": [
+            "sentinel"
+          ],
```

Comment:
Is this going to be a discriminated type? It should specify it. At the very least, I'd expect `kind` is required and should be authored as such.

Reply:
I don't see anything "deriving" from this, so presumably it's not a discriminator. Please be sure to reply to comments before resolving.